tools/db/wipeTransientDBs.sh

```diff
@@ -55,6 +55,20 @@ do
     fi
 done
 
+## drop the API Gateway routes database (recreated via routemgmt.yml)
+DB_APIGW=$(getProperty "$PROPERTIES_FILE" "db.whisk.apigw")
+echo "dropping database: '$DB_APIGW'"
+
+# drop the database
+CMD="$CURL_ADMIN -X DELETE $URL_BASE/$DB_APIGW"
+RES=$($CMD)
+if [[ "$RES" == '{"ok":true}' ]]; then
+    echo DELETED
+else
+    # table may not exist
+    echo WARNING: $RES
+fi
+
```

Comment:
I feel like we should not conflate the standard whisks db with the API gateway db. Any reason we need to delete the DB in this script?